recycle-service/src/main/java/com/handlers/PermanentlyDeleteImageHandler.java

```diff
@@ -0,0 +1,59 @@
+package com.handlers;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.utils.DynamoDBUtils;
+import com.utils.ResponseUtils;
+import com.utils.S3Utils;
+
+import lombok.extern.slf4j.Slf4j;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+
+import java.util.Map;
+
+@Slf4j
+public class PermanentlyDeleteImageHandler implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+    private final String TABLE_NAME = System.getenv("IMAGE_TABLE");
```

Comment:
```suggestion
    private final String tableName = System.getenv("IMAGE_TABLE");
```
I found that this the best naming convention